tasks/actorstate/actorstate_test.go

```diff
@@ -2,6 +2,7 @@ package actorstate_test
 
 import (
 	"context"
+	"github.com/filecoin-project/lily/chain/actors/adt"
```

Comment:
nit: I prefer to see aliases on packages which are imported with a name other than their path.

```
	adt2 "github.com/filecoin-project/lily/chain/actors/adt"
```

Reply:
There are two adt packages here, one from specs-actors which is required for initializing adt strcutures, and a lily specific adt which is required for satisfying the MockAPI interface. I will rename the specs-actors adt package to `sa_adt`